src/components/LoginRegister.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useState } from 'react'
+import login_costume from '../assets/images/login-costume.svg'
+import { Link } from 'react-router-dom'
+import { Button, Form } from 'react-bootstrap'
+
+const URL = 'http://localhost:3000'
```

Comment:
Good for testing, But API is ready. Let's fix it using .env var~~ 🚀
> Can see at https://vitejs.dev/guide/env-and-mode.html Or branch login 
```suggestion
const URL = 'http://localhost:3000' <- should be 3001
```